src/site_scout/site_scout.py

```diff
@@ -481,10 +483,9 @@ def run(
                 url_str = str(next_url)
                 total_visits += 1
                 LOG.info(
-                    "[%02d/%02d] %s %r",
+                    "[%02d/%02d] %r",
                     total_visits,
                     total_urls,
-                    next_url.uid[:8],
                     f"{url_str[:80]}..." if len(url_str) > 80 else url_str,
```

Comment:
Btw, this pattern appears twice in this file. Maybe it should be a util function instead?